ragstack-e2e-tests/pyproject.langchain.toml

```diff
@@ -6,7 +6,7 @@ license = ""
 authors = ["DataStax"]
 
 [tool.poetry.dependencies]
-python = ">=3.9,<3.12"
+python = ">=3.9,<3.12,!=3.9.7"
```

Comment:
trulens depends on streamlit, which specifically disallows 3.9.7. I don't think this is a problem since it's in the `e2e-tests` only?

Reply:
yep no problem